lib/pharos/scripts/kubeadm-renew-certs.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -eu
+
+if [ "$SKIP_UNSET_PROXY" = "true" ]; then
```

Comment:
🤔  I realize this comes from copypaste but I wonder if it's right, because it looks wrong.. `if skip_unset = true { unset }`